sql/drop.go

```diff
@@ -87,3 +88,67 @@ func (p *planner) DropTable(n *parser.DropTable) (planNode, error) {
 	}
 	return &valuesNode{}, nil
 }
+
+// DropDatabase drops a database.
+func (p *planner) DropDatabase(n *parser.DropDatabase) (planNode, error) {
+	if n.Name == "" {
+		return nil, errEmptyDatabaseName
+	}
+
+	nameKey := structured.MakeNameMetadataKey(structured.RootNamespaceID, string(n.Name))
```

Comment:
right. that's probably why we haven't combined them before. Ok, you can leave it, I'll take a look and see how to refactor this.